regression-test/suites/nereids_function_p0/agg_function/agg.groovy

```diff
@@ -1474,6 +1474,183 @@ suite("nereids_agg_fn") {
 	sql '''
 		select /*+SET_VAR(disable_nereids_rules='THREE_PHASE_AGGREGATE_WITH_DISTINCT, TWO_PHASE_AGGREGATE_WITH_DISTINCT')*/ count(distinct id), histogram(kstr) from fn_test'''
 
+	qt_sql_linear_histogram_TinyInt_gb '''
+		select linear_histogram(ktint, 10) from fn_test group by kbool order by kbool'''
+	qt_sql_linear_histogram_TinyInt '''
+		select linear_histogram(ktint, 10) from fn_test'''
+	qt_sql_linear_histogram_TinyInt_offset '''
+		select linear_histogram(ktint, 10, 1) from fn_test'''
+	qt_sql_linear_histogram_TinyInt_agg_phase_1 '''
+		select count(id), linear_histogram(ktint, 10) from fn_test group by id order by id'''
+	qt_sql_linear_histogram_TinyInt_agg_phase_2 '''
+		select count(distinct id), linear_histogram(ktint, 10) from fn_test'''
+	qt_sql_linear_histogram_TinyInt_agg_phase_3 '''
+		select /*+SET_VAR(disable_nereids_rules='THREE_PHASE_AGGREGATE_WITH_DISTINCT, TWO_PHASE_AGGREGATE_WITH_DISTINCT')*/ count(distinct id, kint), linear_histogram(ktint, 10) from fn_test group by kbool order by kbool'''
+	qt_sql_linear_histogram_TinyInt_agg_phase_4 '''
+		select /*+SET_VAR(disable_nereids_rules='THREE_PHASE_AGGREGATE_WITH_DISTINCT, TWO_PHASE_AGGREGATE_WITH_DISTINCT')*/ count(distinct id), linear_histogram(ktint, 10) from fn_test'''
```

Comment:
It seems this test failed from the log:
http://43.132.222.7:8111/buildConfiguration/Doris_DorisRegression_P0Regression/514605?expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandBuildChangesSection=true&expandBuildTestsSection=true&expandPull+Request+Details=true

Tips: You can run the regression-test locally to fix it.

Reply:
Thank you. I just found that I can log in as guest.